### PR TITLE
feat: Support deploying in single user mode with operator

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatoliy Bazko <abazko@redhat.com>

### What does this PR do?
Support deploying in single user mode with operator

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/13420
https://github.com/eclipse/che-operator/pull/185

### CODE REVIEW

File: src/commands/server/start.ts [hunk 1]
```diff
@@ -205,6 +204,7 @@ export default class Start extends Command {
       flags.cheimage && ignoredFlags.push('--cheimage')
       flags.debug && ignoredFlags.push('--debug')
       flags.domain && ignoredFlags.push('--domain')
+      flags.multiuser && ignoredFlags.push('--multiuser')
```
Comment (mmorhun) [inline src/commands/server/start.ts]: Shouldn't chectl flags have higher priority over config file?

Comment (tolusha) [reply src/commands/server/start.ts]: They have a higher priority over defaults but a less priority in comparison with provided CR

File: src/tasks/installers/installer.ts [hunk 2]
```diff
@@ -80,9 +80,8 @@ export class InstallerTasks {
     } else if (flags.installer === 'operator') {
       title = '🏃‍  Running the Eclipse Che operator'
       task = () => {
-        // The operator installs Eclipse Che in multiuser mode only
+        // The operator installs Eclipse Che in multiuser mode by default
```
Comment (AndrienkoAleksandr) [inline src/tasks/installers/installer.ts]: Maybe, make sense to have a flag single user, to have ability fast deploy Che in this mode without playing with CR yaml or CR yaml patch?

Comment (tolusha) [reply src/tasks/installers/installer.ts]: Single-user mode for operator is what I would like to hide.
It is just to deprecate `deploy.sh` script

**Pull request merged**